### DISCUSSION TITLE
Need help with dark theme customization in IG v9.0.5.726

### DISCUSSION THREAD

**Discussion by VitalSkib** [d2phap/ImageGlass#1636] [Q&A] [answered]
Hi there. 
I have a couple of questions and would appreciate any help. 
So, I want to customize the dark Kobe theme, well, just replace the icons and buttons background and I almost did it. 
However, I have a problem with changing the background of the navigation buttons (see 1. in the screenshot). 
Is there any way to change the background of these buttons to a custom one instead of an accent color? 
Couldn't find anything in the configuration files.

And the second question is not related to the first, but.. 
Can you tell me how can I add a `Color Picker` button to the toolbar (just like 2. in the screenshot)?  

Thanks in advance.

![2023_08_26_Screenshot](https://github.com/d2phap/ImageGlass/assets/43848892/937e5052-2f90-4c81-8659-fc89e18a261a)

**Comment by VitalSkib** [comment]
Update. 
I was able to figure out the second question and added a `Color Picker` button that functions as it should. 
However, this does not work with the `Crop` tool. 
It adds a `Crop` button to the toolbar, but gives me an error when I click on that button (see attached image). 
So, what would be the correct name to call the crop tool?
tia

![2023_08_26_Screenshot_2](https://github.com/d2phap/ImageGlass/assets/43848892/620976ec-6a47-4dd9-8890-a1406ba72d70)

**Reply by d2phap** [reply, marked as answer]
It's `MnuCropTool`. You can find all menu names here (starts with `Mnu`): https://github.com/d2phap/ImageGlass/blob/4f8152ae11cb01bbc3f81aa431870fcd85d8cf2f/v9/ImageGlass/FrmMain.Designer.cs#L37-L141

For the navigation arrow buttons, it just follows the accent color

**Comment by VitalSkib** [comment]
> MnuCropTool

Oh thank you very much for the quick reply. It definitely works. 😎👍

As for the background color of the navigation buttons, ok, thanks..
But is there any solution for this?
I mean, is there any way to replace it with another color instead of an accent color?

**Reply by d2phap** [reply]
Currently, there is no option to change the color. I will consider adding it in the next update.

**Comment by VitalSkib** [comment]
> Currently, there is no option to change the color. I will consider adding it in the next update.

Yes, thanks.. 
It's not such a big problem, but I think it would be very useful to have such an option in order to customizing the appearance.
